src/components/SearchGrid.vue

```diff
@@ -29,28 +30,41 @@
       <li><a href="/browse/new">new</a></li>
     </ul>
     <div class="search-grid_ctr" ref="gridItems">
-      <search-grid-cell v-for="(image) in _images"
-        :key="image.id"
-        :image="image"
-        :includeAddToList="includeAddToList" />
+      <div
```

Comment:
Can't this `div` be moved to the `SearchGridCell` component?

Reply:
Let me see if I can work that out alongwith the creation of a new `MasonrySearchGridCell` component.

Reply:
I'm afraid not. This `<div>` refers to the masonry container, and not one cell individually. The `v-for` attribute is on the nested cell component.